tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -422,22 +410,6 @@ def test_it_sets_the_right_javascript_config_settings(
             }
         )
 
-        self._assert_authorization_valid_jwt(
-            form_fields["authorization"],
-            {
-                "oauth_consumer_key": "TEST_OAUTH_CONSUMER_KEY",
-                "roles": "TEST_ROLES",
-                "user_id": "TEST_USER_ID",
-            },
-        )
-
-    def _assert_authorization_valid_jwt(self, authorization, expected_values):
-        assert authorization.startswith("Bearer ")
-        assert (
-            decode_jwt(authorization[len("Bearer ") :], "test_secret")
-            == expected_values
-        )
```

Comment:
We don't care _what_ the authorization string is or whether it decodes successfully as a JWT with a given secret. That's `BearerTokenSchema`'s job, and it has its own unit tests. The fact that the authorization string is a JWT is an internal implementation detail of `BearerTokenSchema`, not part of its public API. In `BasicLTILaunchViews` we only care that it came from `BearerTokenSchema`